domain/application/service/status_test.go

```diff
@@ -4,17 +4,78 @@
 package service
 
 import (
+	"time"
+
 	jc "github.com/juju/testing/checkers"
 	gc "gopkg.in/check.v1"
 
 	"github.com/juju/juju/core/status"
+	coreunit "github.com/juju/juju/core/unit"
 	"github.com/juju/juju/domain/application"
 )
 
 type statusSuite struct{}
 
 var _ = gc.Suite(&statusSuite{})
 
+var now = time.Now()
+
+func (s *statusSuite) TestEncodeCloudContainerStatus(c *gc.C) {
+	testCases := []struct {
+		input  *status.StatusInfo
+		output *application.StatusInfo[application.CloudContainerStatusType]
+	}{
+		{
+			input: &status.StatusInfo{
```

Comment:
Can you put summeries in each of these table driven tests that get printed out in a gc.Commentf when they fail? Printing out the number is a pretty painful way to find which failed.

Either that or change this to not be a table driven test.

Reply:
These particular tests I feel are quite obvious. 

However, there are some table-bases tests later which could do with some explanation